packages/flutter/lib/src/cupertino/colors.dart

```diff
@@ -397,14 +397,14 @@ class CupertinoColors {
   /// [UIColor.tertiarySystemFill](https://developer.apple.com/documentation/uikit/uicolor/3255076-tertiarysystemfill).
   static const CupertinoDynamicColor tertiarySystemFill = CupertinoDynamicColor(
     debugLabel: 'tertiarySystemFill',
-    color: Color.fromARGB(30, 118, 118, 128),
-    darkColor: Color.fromARGB(61, 118, 118, 128),
-    highContrastColor: Color.fromARGB(51, 118, 118, 128),
-    darkHighContrastColor: Color.fromARGB(81, 118, 118, 128),
-    elevatedColor: Color.fromARGB(30, 118, 118, 128),
-    darkElevatedColor: Color.fromARGB(61, 118, 118, 128),
-    highContrastElevatedColor: Color.fromARGB(51, 118, 118, 128),
-    darkHighContrastElevatedColor: Color.fromARGB(81, 118, 118, 128),
+    color: Color.fromRGBO(118, 118, 128, 0.12),
```

Comment:
Are these the same values as the ARGB variant?

Reply:
At your reminder, I rechecked this part of the color and found that in fact the color I got from the capture view hierarchy converted and matched the original color, which was my error and is now resolved.